tests/phpunit/tests/feed/rss2.php

```diff
@@ -117,6 +149,33 @@ public function test_rss_element() {
 		$this->assertCount( 1, $rss[0]['child'] );
 	}
 
+	/**
+	 * Test the <rss> element to make sure its present and populated
+	 * with the modified version of the title.
+	 *
+	 * @ticket 13867
+	 */
+	function test_feed_title() {
+		add_filter( 'comments_feed_title', array( $this, 'apply_comments_feed_title' ), 10, 2 );
+		$this->go_to( '/comments/feed/' );
+		$feed = $this->do_rss2_comments();
+		$xml  = xml_to_array( $feed );
+
+		// Get the feed title.
+		$title = xml_find( $xml, 'rss', 'channel', 'title' );
+		$this->assertSame( 'Filtered Title', $title[0]['content'] );
+	}
+
+	/**
+	 * Apply the comments feed title filter.
+	 *
+	 * @ticket 13867
+	 */
+	function apply_comments_feed_title( $item_title, $the_title_rss ) {
```

Comment:
```suggestion
	public function apply_comments_feed_title( $item_title, $the_title_rss ) {
```
All methods should indicate visibility. As this method is used in as a callback, it should have `public` visibility.